app/gui/view/graph-editor/src/builtin/visualization/java_script/table.js

```diff
@@ -99,7 +99,13 @@ class TableVisualization extends Visualization {
             } else if (params.value === '') {
                 return '<span style="color:grey; font-style: italic;">Empty</span>'
             }
-            return params.value.toString()
+            return params.value
+                .toString()
+                .replace(
+                    /[&<"'>]/g,
+                    m =>
+                        ({ '&': '&amp;', '<': '&lt;', '"': '&quot;', "'": '&#39;', '>': '&gt;' }[m])
+                )
```

Comment:
```suggestion
            const mapping = { '&': '&amp;', '<': '&lt;', '"': '&quot;', "'": '&#39;', '>': '&gt;' };
            return params.value
                .toString()
                .replace(
                    /[&<"'>]/g,
                    m => mapping[m]
                )
```

I'm not sure how good the JS optimizer is, but it feels like it will have an easier job if we initialize this map only once and not on each callback-call (which can be a lot if some huge HTML is being escaped).

Also, I think we should move this into a separate function:

```
function escapeHTML(code) { ... }
```

It will make the code more readable.